README.md

```diff
@@ -55,32 +55,38 @@ This extension contributes the following settings:
 * `ConEmu.reuseInstance`: open console in existing instance or always new window
 * `ConEmu.showTitlebarIcon`: show button in title bar (default: false)
 * `ConEmu.showMessageInOutputPanel`: show messages in output panel (default: false)
+* `ConEmu.useCustomConEmuRunOption`: use [ConEmu -Run option](https://conemu.github.io/en/ConEmuArgs.html) to launch predefined console (default: false)
```

Comment:
I think you forgot to remove this redundant flag from the README

Reply:
Thanks, I removed this.